src/scicookie/{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -142,4 +147,20 @@ multi_line_output = 3
 include_trailing_comma = true
 skip_glob = ["docs/*", "*.egg-info"]
 {% endif %}
+{%- if cookiecutter.use_bandit == "yes" %}
+[tool.bandit]
+exclude_dirs = ["tests"]
+targets = "{{ package_path }}"
+{% endif %}
+{%- if cookiecutter.use_vulture == "yes" %}
+[tool.vulture]
+exclude = ["tests"]
+ignore_decorators = []
+ignore_names = []
+make_whitelist = true
+min_confidence = 80
```

Comment:
@xmnlab I think we can change 80 by 60, some days ago @YurelyCamacho and I was playing with a function and the vulture show a message when you create a function but don't use it

Reply:
yep, in this case I just decided to move it up to 80 because with 60% it returns a bunch of false positive :(

but maybe it is missing a more curated configuration for the projects I tested it

and the users actually can manually change it if they want .. but the idea is to provide them something that will not make their life a hell XD

but we can explain more about that in the documentation https://github.com/osl-incubator/scicookie/blob/0.1.0/docs/guide.md
For example, we could add to the documentation information like, some recommendations, how to use it, some parameters to use, etc

maybe something around 3 or 4 paragraphs